### ISSUE TITLE
Be more tolerant of certain… less appropriate feed mimetypes?

### ISSUE THREAD

**Issue by Frenzie** [FreshRSS/FreshRSS#1410]
> A feed could not be found at `http://www.oapen.org/latesttitles.rss`; the status code is `200` and content-type is `application/x-download` [http://www.oapen.org/latesttitles.rss]

I imagine it would need some adjustment in https://github.com/Alkarex/FreshRSS/blob/a37484fd2fee58d48d8e58535b34d8c63352c51e/lib/SimplePie/SimplePie/Content/Type/Sniffer.php. QuiteRSS has no trouble with the feed.

**Comment by Alkarex** [comment]
Hello @Frenzie ,
Could you try to add the magic `#force_feed` suffix to the feed URL?

**Comment by Frenzie** [comment]
That does the trick! I didn't know about that. Perhaps it could be offered as a suggestion to the user? "A feed could not be found, blablabla, would you like to try interpreting it as a feed anyway?"

**Comment by Alkarex** [comment]
I am glad it works. Yes indeed, this option could be advertised a bit more. Patch welcome, if you feel like it :-)

**Comment by Frenzie** [comment]
@Alkarex 
This is something I failed to notice previously, but my FreshRSS hasn't updated the feed in question for about a month since my testing the update to 1.7. The last retrieved entry was from May 25th, which clearly fits given that #1548 was opened on May 26th. The error is the same as before.

```
2017-06-26 22:23:56A feed could not be found at `http://www.oapen.org/latesttitles.rss`; the status code is `200` and content-type is `application/x-download` [http://www.oapen.org/latesttitles.rss]
```

Manually adding (or re-adding?) `#force_feed` fixed it, but to my knowledge I never removed it. I don't normally go looking at or messing with feed settings unless a problem pops up, after all.

**Comment by Alkarex** [comment]
Did you notice whether the force_feed hash was gone before you added it
back? Maybe a bug e.g. if the feed got a new address (e.g. via a 301). Do
you think the problem happened during update? There were a few SimplePie
updates in FreshRSS 1.7, but I cannot immediately think of a precise
problem, though.

**Comment by Frenzie** [comment]
When I said adding or re-adding I was talking about changing the URL in "manage", sorry if I caused any confusion. The URL was simply `http://www.oapen.org/latesttitles.rss` and the feed was refusing to update with that error message until I added the hash. 

What I don't remember and probably never checked is whether the URL has always been this way (i.e., whether `#force_feed` was only used to force *adding*) or whether the update somehow got rid of it.
